client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -86,7 +86,7 @@ export const DateUtils = {
     differenceInDays(Date.now(), dateInputHandler(date)),
   startOfDay,
   startOfYear,
-  formatRFC3339,
+  formatRFC3339: (date: Date) => (isValid(date) ? formatRFC3339(date) : null),
```

Comment:
is there a reason to return `null`? it would simplify the usage if you could return `undefined` instead 😄

Reply:
Yeah, true. 45ff8470d